src/test/java/com/rackspace/jenkins_nodepool/Mocks.java

```diff
@@ -100,8 +102,11 @@
     List<Attempt> attemptListSuccess;
     Attempt attemptSuccess;
 
+
     public Mocks() {
         requestTimeout = 30;
```

Comment:
The default of 30 seconds is quite short, given that hardware nodes take up to 10 minutes to be built and available to hand over to jenkins.

Reply:
This is for unit tests only, won't be used for any real builds.